sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts

```diff
@@ -246,7 +251,8 @@ class GapiManager {
 
   private static _loadBigQuery(): Promise<void> {
     return this.loadGapi()
-      .then(() => gapi.client.load('bigquery', 'v2'));
+      .then(() => gapi.client.load('bigquery', 'v2'))
```

Comment:
It is necessary. Without this line, gapi.client.bigquery does not get filled, so calls such as to gapi.client.bigquery.projects.list fail.